auth/login.html

```diff
@@ -4,40 +4,65 @@
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>로그인</title>
+
+    <!-- favicon 불러오기 -->
     <link rel="icon" type="image/svg" href="/image/logo.svg" />
+
+    <!-- Pretendard 폰트 불러오기 -->
     <link
       rel="stylesheet"
       as="style"
       crossorigin
       href="https://cdn.jsdelivr.net/gh/orioncactus/pretendard@v1.3.9/dist/web/static/pretendard.min.css"
     />
+    <!-- normalize css 불러오기 -->
     <link
       rel="stylesheet"
       href="https://cdnjs.cloudflare.com/ajax/libs/normalize/8.0.1/normalize.min.css"
     />
-    <link rel="stylesheet" href="/css/common.css">
-    <link rel="stylesheet" href="/css/auth.css" />
+
+    <!-- css 불러오기 -->
+    <link rel="stylesheet" href="/css/reset.css" />
+    <link rel="stylesheet" href="/css/common.css" />
+    <link rel="stylesheet" href="/auth/auth.css" />
+
+    <!-- script 불러오기 -->
+    <script defer type="module" src="/auth/common.js"></script>
```

Comment:
❗️ 수정요청
module js의 경우 자동으로 defer 되기 때문에 defer 속성을 빼셔도 됩니다~

```suggestion
    <script type="module" src="/auth/common.js"></script>
```

[mdn script](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/script#defer)